### PR TITLE
draft for #423

### PR DESCRIPTION
**Pull request by bennyvasquez**
This is a draft of the potential change I suggested in #423

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -30,6 +30,62 @@ module.exports = {
...
+            },
+            {
+            title: "Older AlmaLinux 8 releases",
+  			  children: [
+              '/release-notes/8.8',
+              '/release-notes/8.7',
+              '/release-notes/8.6',
+              '/release-notes/8.5-ppc',
+              '/release-notes/8.5',
+              '/release-notes/8.5-beta-ppc',
```
Comment (sboldyreva) [inline docs/.vuepress/config.js:68]: I thinks we should put 8.5-beta-ppc to betas

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -30,6 +30,62 @@ module.exports = {
...
+	            title: "AlmaLinux 8 betas",
+	  			  children: [
+                   '/release-notes/8.9-beta',
+                   '/release-notes/8.8-beta',
+                   '/release-notes/8.7-beta',
+                   '/release-notes/8.6-beta',
+                   '/release-notes/8.5-beta',
+                   '/release-notes/8.4-beta-arm',
+                   '/release-notes/8.4-beta',
+                   '/release-notes/8.3-rc',
```
Comment (sboldyreva) [inline docs/.vuepress/config.js:82]: and this (8.3-rc) looks like should be in stables

File: docs/.vuepress/config.js (hunk 1 above)
Comment (bennyvasquez) [reply docs/.vuepress/config.js:68]: Good catch!

File: docs/.vuepress/config.js (hunk 2 above)
Comment (bennyvasquez) [reply docs/.vuepress/config.js:82]: RC = release candidate, which isn't quite stable to me. Happy to move it if you feel differently, though!

**Review by codyro — APPROVED** [review]

**Pull request merged**